src/main/java/frc/utils/devicewrappers/SparkMaxConfiguration.java

```diff
@@ -0,0 +1,85 @@
+package frc.utils.devicewrappers;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.utils.PIDObject;
+
+public class SparkMaxConfiguration {
+
+	public double conversionFactor;
```

Comment:
make a conversion factor for position and velocity. sometimes, you measure 2 things when velocity and distance (like in swerve, you add the position conversion factor later.)
add another parameter for PositionConversionFactor and VelocityConversionFactor